dbt/adapters/teradata/connections.py

```diff
@@ -272,6 +275,46 @@ def open(cls, connection):
             def connect():
                 connection.handle = teradatasql.connect(**kwargs)
                 connection.state = 'open'
+
+                # checking if query_band connection parameter exists. If it does -> executing the set query_band sql
+                if credentials.query_band:
+                    try:
+                        cur = connection.handle.cursor()
+                        query_band_text = credentials.query_band
+
+                        if query_band_text[-1] != ';':
+                            query_band_text += ';'
+
+                        """checking if appname= key exist in query_band and if dbt exist as value of 
+                        that key otherwise appending DBT in the end """
+                        if 'appname=' in query_band_text or 'appname =' in query_band_text:
+                            pair = query_band_text.split(';')
+                            for i in range(len(pair)):
+                                if 'appname' in pair[i]:
+                                    val = pair[i].split('=')[1]
+                                    patterns = "^" + re.escape('dbt') + "$"
+                                    if not re.match(patterns, val):
+                                        pair[i] += '+dbt'
+                            query_band_text = ';'.join(pair)
+                        else:
+                            query_band_text = query_band_text + 'appname=dbt;'
+
+                        if 'org=' not in query_band_text and 'org =' not in query_band_text:
```

Comment:
Got it here. If org not provided in query band, then we are appending default one. If not it will use user provided value for org.

Reply:
Yes Satish